### ISSUE TITLE
Pipe stdout AND stderr from commands into tee (and then into log files)

### ISSUE THREAD

**Issue by TommyMurphyTM1234** [gnu-mcu-eclipse/openocd-build#6]
Small enhancement suggestion - change all occurrences of "some-command | tee some-log-file" to "some-command 2>&1 | tee some-log-file" to ensure that stdout AND stderr messages are captured to facilitate diagnosing build issues/failures.
See below for a list of places where this idiom occurs.

```
user@robin:~/Downloads/openocd-build.git/scripts$ find . -name "*.sh" -print -exec grep tee {} \;
./defs-source.sh
./container-build.sh
./build.sh
./container-openocd-functions-source.sh
        ) | tee "${INSTALL_FOLDER_PATH}"/configure-openocd-output.txt
      ) | tee "${INSTALL_FOLDER_PATH}"/make-newlib-output.txt
./helper/build-helper.sh
./helper/container-functions-source.sh
./helper/host-functions-source.sh
./container-lib-functions-source.sh
        ) | tee "${INSTALL_FOLDER_PATH}"/configure-libusb1-output.txt
      ) | tee "${INSTALL_FOLDER_PATH}"/make-libusb1-output.txt
        ) | tee "${INSTALL_FOLDER_PATH}/configure-libusb0-output.txt"
      ) | tee "${INSTALL_FOLDER_PATH}/make-libusb0-output.txt"
      ) | tee "${INSTALL_FOLDER_PATH}/make-libusb-w32-output.txt"
      ) | tee "${INSTALL_FOLDER_PATH}/configure-libftdi-output.txt"
      ) | tee "${INSTALL_FOLDER_PATH}/make-libftdi-output.txt"
        ) | tee "${INSTALL_FOLDER_PATH}/configure-libiconv-output.txt"
      ) | tee "${INSTALL_FOLDER_PATH}/make-libiconv-output.txt"
        ) | tee "${INSTALL_FOLDER_PATH}/configure-hidapi-output.txt"
        ) | tee "${INSTALL_FOLDER_PATH}/make-hidapi-output.txt"

```

**Comment by ilg-ul** [comment]
Thank you, good idea.

However, please note that I never had to check the saved logs while developing the scripts. During development the recommended solution is to use `--develop`, which allows to restart the script at any moment, even when running inside Docker.

**Comment by TommyMurphyTM1234** [comment]
Thanks - I can't remember the details but I had a case recently when using the script and the build failed for some reason and until I modified it to combine stdout and stderr I was unable to see what was wrong. However maybe --develop would have helped but I forgot about that option until you mentioned it again above! :-)

**Comment by ilg-ul** [comment]
Fixed since 0.10.0-9.
